pkg/cloudprovider/vsphereparavirtual/controllers/routablepod/ipaddressallocation/ipaddressallocation_controller.go

```diff
@@ -0,0 +1,277 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipaddressallocation
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	nsxapisv1alpha1 "github.com/vmware-tanzu/nsx-operator/pkg/apis/nsx.vmware.com/v1alpha1"
```

Comment:
Just like `vmopv1`, we change this to `nsxapisv1` & `nsxinformerv1` ande etc.?

Reply:
Unlike vmop, nsx v1alpha1 and v1alpha2 are totally different APIs and not compatible at all.

Reply:
I will rename it to v1 since v1alpha2 was removed in nsx-operator just now.